arrays.py

```diff
@@ -192,4 +192,5 @@
     "a parallel play session",
     "a new stim toy",
     "some peace and quiet",
+    "a pipe bomb"
```

Comment:
```suggestion
    "a pipe bomb",
```

Tox gets upset without trailing commas